src/base/camera.cc

```diff
@@ -96,6 +96,10 @@ double Camera::FocalLength() const {
   return params_[idxs[0]];
 }
 
+double Camera::NormalizedFocalLength() const {
```

Comment:
Please add a unit test for this ``base/camera_test.cc``, same for the new methods below.

Reply:
Done